### PR TITLE
Pom updates

### PR DESCRIPTION
**Pull request by markt-asf**
Updates in preparation for first release

### CODE REVIEW

File: api/pom.xml [hunk 1]
```diff
@@ -19,20 +19,21 @@
...
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+
     <parent>
-        <groupId>net.java</groupId>
-        <artifactId>jvnet-parent</artifactId>
-        <version>5</version>
+        <groupId>org.eclipse.ee4j</groupId>
+        <artifactId>project</artifactId>
+        <version>1.0.1</version>
```
Comment (joakime) [inline api/pom.xml]: shouldn't this be `1.1.1` too?

Comment (markt-asf) [reply api/pom.xml]: That is the version of the EE4J parent. 1.0.1 is the latest.

Comment (joakime) [reply api/pom.xml]: oh.

**Comment by markt-asf** [comment]
Looks like I've broken something in the Javadoc generation. Looking now...

**Comment by joakime** [comment]
Looks like JSONP has that glassfish spec check plugin enabled and configured.

https://github.com/eclipse-ee4j/jsonp/blob/EE4J_8/api/pom.xml#L82-L103

**Comment by markt-asf** [comment]
They haven't changed their groupId yet...

**Comment by joakime** [comment]
Perhaps we shouldn't change our groupId then too? (boy this is confusing)

**Comment by markt-asf** [comment]
It is looking like the groupId change is going to have to happen. In some ways, that we tried it and found the problems early is good. I see several options:

1. Wait. Revisit this PR once the groupId question is resolved
2. Apply the PR as is
3. Rework the PR minus the groupId and spec plug-in check changes. Put those changes in a separate PR and leave it open until the groupId quesiton is resolved.

I'm leaning towards 3. WDYT?

**Comment by joakime** [comment]
Right, I think option 3 is a good choice.
We can bring in @m0mus (and others) on the new PR specific to the groupId to keep the conversation focused.

**Pull request merged**